Source/JavaScriptCore/runtime/ArrayConstructor.cpp

```diff
@@ -43,7 +43,7 @@ const ClassInfo ArrayConstructor::s_info = { "Function"_s, &InternalFunction::s_
 /* Source for ArrayConstructor.lut.h
 @begin arrayConstructorTable
   of        JSBuiltin                   DontEnum|Function 0
-  from      JSBuiltin                   DontEnum|Function 0
+  from      JSBuiltin                   DontEnum|Function 1
```

Comment:
This isn't actually necessary as the `length` of `JSBuiltin` functions is deducted from their signature. Covered by test262.

Reply:
Yes, but anyway, I just fixed it since it is wrong :)